sprints/2024/02_08/ztf_bts-ngc.ipynb

```diff
@@ -3,9 +3,7 @@
   {
```

Comment:
<p>I think you need to enable pre-commit to clean outputs out, run <code>pre-commit install</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/notebooks_lf/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>